src/lib/components/identity/identity-details.svelte

```diff
@@ -57,30 +87,31 @@
                 <div class="text-secondary text-break">
                     <span class="fw-bold">Creator: </span><span class="text-break ">{identity?.creator || 'unknown'}</span>
                 </div>
+                <div class="text-secondary text-break">
+                    <span class="fw-bold">Role: </span><span class="text-break ">{identity?.role || 'unknown'}</span>
+                </div>
             </div>
             <div class="d-flex flex-column align-items-start">
                 {#if actionButtons}
                     {#each actionButtons as { label, onClick, icon, color, loading, disabled, hidden }}
-                        {#if !hidden}
```

Comment:
Why did you remove this if? Now the 'Add credential' button is shown to even non-admin users.